dev-docs/bidders/smilewanted.md

```diff
@@ -2,7 +2,7 @@
 layout: bidder
 title: Smile Wanted
 description: SmileWanted Bidder Adapter
-media_types: banner, video
+media_types: banner, native, video
```

Comment:
Please add 

- multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
- safeframes_ok: true / false

Meta data to help publishers vet your adapter.
See https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter

Reply:
Thank you for the review.
The metadatas are up-to-date.